### PR TITLE
Define const for differents paths

### PR DESCRIPTION
**Pull request by gwendalF**
#47

### CODE REVIEW

File: src/index.rs [hunk 1]
```diff
@@ -802,7 +803,7 @@ mod tests {
 
             assert_eq!(actual.collisions.len(), 0);
             assert_eq!(actual.size(), 2);
-
+            #[cfg(target_os = "unix")]
             file.set_permissions(Permissions::from_mode(0o222))
                 .expect("Should be fine");
```
Comment (kirillt) [inline src/index.rs:808]: Curious, what is it about?

**Review by kirillt — DISMISSED** [review]
Almost good, one thing to fix @gwendalF

File: src/index.rs (hunk 1 above)
Comment (gwendalF) [reply src/index.rs:808]: It's using the PermissionExt https://doc.rust-lang.org/std/os/unix/fs/trait.PermissionsExt.html which is only available on unix platforms

File: src/lib.rs [hunk 2]
```diff
@@ -21,13 +21,22 @@ use log;
 
 pub const STORAGES_FOLDER: &str = ".ark";
 
-// allowed to be lost (cache)
-pub const INDEX_PATH: &str = "index";
-pub const PREVIEWS_PATH: &str = "previews";
-
 // must not be lost (user data)
 pub const METADATA_PATH: &str = "meta";
 pub const TAGS_PATH: &str = "tags";
```
Comment (kirillt) [inline src/lib.rs:26]: These two must be replaced by `METADATA_STORAGE_FOLDER` and `TAG_STORAGE_FILE` correspondingly.

File: src/lib.rs [hunk 3]
```diff
@@ -21,13 +21,22 @@ use log;
 
 pub const STORAGES_FOLDER: &str = ".ark";
```
Comment (kirillt) [inline src/lib.rs:22]: Let's rename to `ARK_FOLDER`.

Comment (gwendalF) [reply src/lib.rs]: Done

File: src/index.rs (hunk 1 above)
Comment (kirillt) [reply src/index.rs:807]: Ah, I haven't looked at name of the test.. All is clear now.

**Review by kirillt — APPROVED** [review]

**Pull request merged**